src/app/models/todo.model.ts

```diff
@@ -0,0 +1,6 @@
+export class Todo {
+  name: ArrayBuffer;
+  constructor(user: any) {
```

Comment:
Consider using `LooseObject`. `any` should almost never be used while writing code in Typescript